uftrace.c

```diff
@@ -912,7 +912,8 @@ static void parse_opt_file(int *argc, char ***argv, char *filename, struct opts
 		opts->idx = 0;
 	}
 
-	argp_parse(&file_argp, file_argc, file_argv,
+	if ( file_argv) /* workaround for when the file is empty */
+		argp_parse(&file_argp, file_argc, file_argv,
```

Comment:
We don't add a space after open parenthesis.  Also you'd better use a pair of curly braces when the body of if statement is multi-line.  Please indent below lines as well.

Reply:
This is also relevant when input is empty, i.e `""` or `"  "` or any number of spaces. So i guess i shall leave it out.